.github/workflows/e2e-tests.yml

```diff
@@ -4,12 +4,23 @@ on:
   pull_request:
     paths:
       - 'e2e/**'
+
   workflow_dispatch:
 
 jobs:
 
+  oisy-backend-wasm:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
Do you need this one? I see you do it already in `actions/oisy-backend`

Reply:
Yes, can be removed. Done.